companion/src/modeledit/expodialog.cpp

```diff
@@ -240,7 +240,7 @@ void ExpoDialog::fmInvertAll()
 {
   lock = true;
   for (int i=0; i<CPN_MAX_FLIGHT_MODES; i++) {
-    cb_fp[i]->setChecked(not cb_fp[i]->checkState());
+    cb_fp[i]->setChecked(!cb_fp[i]->checkState());
```

Comment:
I can change it in my pull request if it won't hurt anything else.  It's not my code and I don't know qt.  I was just fixing the busted VS15 build